variables.py

```diff
@@ -3,7 +3,7 @@
 from json import loads
 load_dotenv()
 
-PAGE_PER_SCAN = 5
+PAGE_PER_SCAN = 1
 MAX_WORKERS = 10
```

Comment:
REDIS INTERFACE.py:
1. add error handling- if a request to the redis failed (reading/writing), you should raise error and figure out in code how to handle it.
2. add ftsearch searches. you should index a relevant columns you want to query the data afficiently.